ads/google/a4a/utils.js

```diff
@@ -137,8 +137,7 @@ function buildAdUrl(
   const viewportRect = viewportFor(global).getRect();
   const iframeDepth = iframeNestingDepth(global);
   const dtdParam = {name: 'dtd'};
-  const allQueryParams = [
```

Comment:
@bobcassels had to remove the spread operator as the code closure compiler generates isnt optimal. does the order matter here or can we just stick queryParams and unboundedQueryParams in the head of the array?